src/dateUtils.js

```diff
@@ -0,0 +1,44 @@
+const { DateTime, Duration } = require('luxon');
+
+const toDate = val => {
+  if (DateTime.isDateTime(val)){
+    return val;
+  } else if (typeof val === 'number'){
```

Comment:
```
if () {
  return 
}

if () {
  return
}
```

i think the `else` conditions just add noise and aren't needed here

Reply:
Updated